canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -68,12 +68,17 @@ $toolbar-divider-width: 1px;
 	&.ghost {
 		& button {
 			background-color: $ui-01;
+			padding: 0;
+			.toolbar-item-content {
+				padding: 11px;
```

Comment:
I dont see this padding being applied. We should also stay away from hardcoded values, try to use Carbon tokens instead